core/src/main/scala/org/allenai/common/MathUtils.scala

```diff
@@ -0,0 +1,10 @@
+package org.allenai.common
+
+/** Various convenient utilities for math operations. */
+object MathUtils {
+
+  /** Round a Double to k decimal digits. */
```

Comment:
Done.  Taking `BigDecimal.RoundingMode.Value` as an optional parameter.  OK or would it be better to define our own `BigDecimal`-independent rounding type?